structs.go

```diff
@@ -304,16 +304,16 @@ type Game struct {
 // UnmarshalJSON unmarshals json to Game struct
 func (g *Game) UnmarshalJSON(bytes []byte) error {
 	temp := &struct {
-		Name string          `json:"name"`
+		Name json.Number     `json:"name"`
```

Comment:
This will fail on actual strings though? I think some RawMessage shenanigans is whats needed.